README.md

```diff
@@ -1,7 +1,9 @@
-# bls12_381 [![Crates.io](https://img.shields.io/crates/v/bls12_381.svg)](https://crates.io/crates/bls12_381) #
+![Build Status](https://github.com/dusk-network/bls12_381/workflows/Continuous%20integration/badge.svg)
```

Comment:
Looks like a nit on the syntax of this first line

Reply:
where? I really don't see it